app/dot/cpp/repository/repository/BaseRepository.java

```diff
@@ -246,4 +248,11 @@ protected Aggregation<T> getHistoryAggregation(Filter filter) {
                 .getCollection(getEntityType().getSimpleName() + "_history", getEntityType()));
     return filter != null ? historyAggregation.match(filter) : historyAggregation;
   }
+
+  @NotNull
+  public Aggregation<T> getFindAggregation(String id, long timestamp) {
```

Comment:
I find really hard to get "get find". Can't we just use simple getAggregation or getAggregationById?